worker/echo/meson.build

```diff
@@ -13,3 +13,7 @@ configure_file(
   install: true,
   install_dir: dbus.get_variable(pkgconfig: 'system_bus_services_dir')
 )
+
+install_data('com.redhat.Yggdrasil1.Worker1.echo.conf',
+  install_dir: join_paths(dbus.get_variable(pkgconfig: 'datadir'), 'dbus-1', 'system.d')
```

Comment:
Trying to install this in a RHEL9.2 (also in RHEL8.7)
I couldn't see this files installed:

```
 cat meson-logs/install-log.txt | grep "Worker1"
/usr/share/dbus-1/interfaces/com.redhat.Yggdrasil1.Worker1.xml
```

Reply:
Did you configure the project to install the examples? `meson setup -Dexamples=True`. By default, it does not install the example `echo` worker.

Reply:
I did not, let me check it with that option.